content/concepts/introduction/protocols/ping.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Ping"
+description: "The ping protocol is a simple request response protocol."
+weight: 20
+---
+
+## Ping in libp2p
+
+ICMP Ping is a network utility that can be used as a simple liveness check
+for peers to quickly see if another peer is online and measure RTT.
+
+The libp2p ping protocol is different from the ping command line utility
+([ICMP ping](https://en.wikipedia.org/wiki/Internet_Control_Message_Protocol)),
+as it requires an already established libp2p connection.
+
+A peer opens a stream, sends a request with a payload of 32 random
+bytes, and the receiver responds with 32 bytes on the same stream.
+Peers can reuse a strean for future pings.
+
+The ping protocol ID is `/ipfs/ping/1.0.0`.
+
+### Example
+
```

Comment:
Can you call out here that this is what you get when you use kubo?
As a library, libp2p itself doesn't have a command line interface.